packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.css

```diff
@@ -25,5 +25,5 @@ a {
 }
 
 .empty-list-message {
-  color: #8e959d;
+  color: var(--lwc-colorTextIconDefault, #747474);
```

Comment:
Given we're giving up on ootb localdev support is there a way to define css variables for all of these color values instead of using the hex code every time? Would help avoid mismatches.

Reply:
I think if we start wanting to do this, we should do this everywhere :D not just for the changes I'm making.
We have colors everywhere, we would need to find the right way to include CSS vars across all components.